src/elements/emby-slider/emby-slider.js

```diff
@@ -203,14 +203,14 @@ function setMarker(range, valueMarker, marker, valueProgress) {
 }
 
 function updateMarkers(range, currentValue) {
-    if (range.getMarkerInfo) {
+    if (range.getMarkerInfo && !range.markerInfo) {
```

Comment:
Isn't `markerInfo` (as well as the slider itself) preserved when start playing the next video, so the markers won't be updated?

Reply:
In my testing it was not.

Reply:
I only get to the breakpoint once per refresh. So I assume they are preserved (because videoosd doesn't reload for the next item).

Reply:
hmm yeah you are right... I'm not sure how it worked when I originally tested... https://github.com/jellyfin/jellyfin-web/pull/5601